test_suite/format.jsonnet

```diff
@@ -189,6 +189,7 @@ std.assertEqual(std.format('%-12.4f', [910.3]), '910.3000    ') &&
 std.assertEqual(std.format('%#.0f', [910.3]), '910.') &&
 std.assertEqual(std.format('%#.0f', [910]), '910.') &&
 std.assertEqual(std.format('%.3f', [1000000001]), '1000000001.000') &&
+std.assertEqual(std.format('%f', [-0.1]), '-0.100000') &&
```

Comment:
I would also add `std.format('%d', [-0.1])'. The result will be '-0', I think. Not sure if that's ok.

Reply:
I think it's not OK because it's different to what Python does.